config/checkstyle/checkstyle.xml

```diff
@@ -1,415 +1,363 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!--
-    # Copyright 2015 WSO2 Inc. (http://wso2.org)
-    #
-    # Licensed under the Apache License, Version 2.0 (the "License");
-    # you may not use this file except in compliance with the License.
-    # You may obtain a copy of the License at
-    #
-    # http://www.apache.org/licenses/LICENSE-2.0
-    #
-    # Unless required by applicable law or agreed to in writing, software
-    # distributed under the License is distributed on an "AS IS" BASIS,
-    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-    # See the License for the specific language governing permissions and
-    # limitations under the License.
--->
-
+<?xml version="1.0"?>
```

Comment:
Updated checkstyle to a newer version that supports Java 16. I copy/pasted in the current Google java style config: https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml

Reply:
As a result I had to reformat a ton of code to pass some of the style checks, specifically around indentation.